emails/templates/emails/wrapped_email.html

```diff
@@ -12,110 +12,178 @@
+    <table width="100%" bgcolor="#3D3D3D" style="background: #3D3D3D; padding: 12px 25px; margin-top: 30px; margin-bottom: 30px; width: 96%; border-radius: 6px; max-width: 1200px;" align="center">
       <tr>
-        <td align="center" valign="top" width="100%" style="max-width:800px; padding-top: 0px; padding-bottom: 0px;">
-          <table width="100%" style="max-width:700px; text-align: center; border-collapse: collapse;">
-            <tr>
-              <td width="100%" align="center" style="line-height: 150%;">
-                <p style="color: #363959; font-size: 13px; font-family: sans-serif; margin-top: 0; margin-bottom: 0;">
-                  {% bold_violet_text display_email as bolded_email %}
-                  {% bold_violet_link SITE_ORIGIN 'Firefox Relay' as linked_origin %}
-                  {% ftlmsg 'forwarded-email-header-from' display_email=bolded_email linked_origin=linked_origin as email_wrap_forwarded_from %}
-                  {{ email_wrap_forwarded_from|safe }}
-                  {% if has_attachment %}
-                      {% ftlmsg 'nav-faq' as faq_text %}
-                      {% with SITE_ORIGIN|add:"/faq/" as faq_url %}
-                          {% bold_violet_link faq_url faq_text as faq_link %}
-                          {% ftlmsg 'forwarded-email-header-attachment' faq_link=faq_link size='10' unit='MB' as forwarded_email_wrap_attachment %}
-                          {{ forwarded_email_wrap_attachment|safe }}
-                      {% endwith %}
-                  {% else %}
-                    {% if survey_text and survey_link %}
-                    <br><a href="{{ survey_link }}" target="_blank" rel="noopener noreferrer" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">{{ survey_text }}</a>.
+          <td align="center" valign="top" width="100%" style="padding-top: 0px; padding-bottom: 0px;">
+          <table width="100%" style="max-width:1200px; text-align: center; border-collapse: collapse;">
+              <tr> 
+                <td class="header-block" width="50%" align="left" style="padding-top: 5px;"> 
+                  <img width="30" src="{{ SITE_ORIGIN }}/static/images/email-images/relay-icon.png" style="margin-right: 5px; display: inline-block; vertical-align: top;" alt="relay icon"/>   
+                   
+                  <p style="margin-top: 0; margin-bottom: 5px; display: inline-block;"> 
+                    {% with display_email|striptags|urlencode as mask_url %}
+                    <span style="display: block;"> 
+                    {% ftlmsg 'relay-email-forwarded-from' %} 
+                    <a class="container-link" href="{{ SITE_ORIGIN }}/accounts/profile/#{{ mask_url }}" style="margin-right: 30px;"> {{ display_email|striptags }} </a>
```

Comment:
Embedded and external CSS should be avoided in email templates for several reasons:

Email clients have limited support for CSS: Unlike modern web browsers, most email clients have limited support for CSS. This means that some CSS properties may not render properly or may be ignored altogether, resulting in inconsistent or broken email layouts. Inline CSS, on the other hand, is widely supported by email clients and is the most reliable way to style email templates.

Email clients may strip external CSS: Many email clients, including Gmail and Outlook, automatically strip external CSS from email messages. This means that any styles defined in an external CSS file may not be applied to the email, resulting in an unstyled or poorly styled email layout.

Security concerns: External CSS files can pose security risks in email templates, as they may contain links to external resources that can be used to track user activity or gather personal information. Inline CSS is considered more secure, as it does not rely on external resources.

Inconvenient for recipients: External CSS files require recipients to download additional resources, which can be inconvenient for users on slow or unreliable connections. Inline CSS eliminates this issue, as all styling is included directly in the email message.

Overall, while it may be more convenient to use external or embedded CSS in email templates, it is generally recommended to use inline CSS instead to ensure consistent rendering across different email clients and reduce the risk of security concerns.